configure.ac

```diff
@@ -185,13 +185,14 @@ AC_C_INLINE
 # Checks for libraries.
 AC_CHECK_LIB([pthread], [pthread_create], [AC_SUBST([pthread_LIBS], [-lpthread])], [AC_MSG_ERROR([kronosnet requires pthread library])])
 AC_CHECK_LIB([m], [ceil], [AC_SUBST([m_LIBS], [-lm])], [AC_MSG_ERROR([kronosnet requires m library])])
-AC_CHECK_LIB([rt], [clock_gettime], [AC_SUBST([rt_LIBS], [-lrt])], [AC_MSG_ERROR([kronosnet requires rt library])])
-
-AC_SEARCH_LIBS([dlopen], [dl dld], [dllibs=$ac_res], [AC_MSG_ERROR([kronosnet requires dl library])])
-
-if test "x$dllibs" != "xnone required"; then
-	AC_SUBST([dl_LIBS], [$dllibs])
-fi
+saved_LIBS="$LIBS"
+LIBS=
+AC_SEARCH_LIBS([clock_gettime], [rt], , [AC_MSG_ERROR([clock_gettime not found])])
+AC_SUBST([rt_LIBS], [$LIBS])
+LIBS=
```

Comment:
you can use check_lib_no_libs() to do this work clean. it´s just a few lines above the current dlopen/rt lib check.

Reply:
Yeah, I noticed it, but I found no current uses of it and found it too long-winded for very little gain: the optional arguments of `AC_CHECK_LIB` work well in most cases. And here the point is using `AC_SEARCH_LIBS` instead, because the rt library isn't actually needed on modern systems.
So I'd recommend removing `check_lib_no_libs()` altogether.